src/components/capture/creation/index.tsx

```diff
@@ -104,6 +106,16 @@ function NewCaptureModal() {
     // TODO Schema Editor
     //const [availableSchemas, setAvailableSchemas] = useState<any[]>([]);
 
+    const exitModal = () => {
+        if (schemaFromEditor) {
+            removeSchema();
+        }
+
+        cleanUp();
```

Comment:
The corresponding selector was renamed to `resetState` and I feel this constant should follow suit. Given the number of states accessed by this component, it may be beneficial to indicate which state is being reset: `resetCaptureCreationState`.

The generically named selector does not pose an issue because it is adequately typed and fed to `useCaptureCreationStore()`. However, I believe a stand alone `cleanUp()` or `resetState()` verges on being too ambiguous deeper within the code. Thoughts?

Reply:
I thought I renamed this when we were on a call - I can change it.

Again I don't think it is an issue as it is the state of the  thing you're in.